spec/class.dd

```diff
@@ -712,8 +712,9 @@ $(GNAME Destructor):
         }
         ------
 
-        $(P There can be only one destructor per class, the destructor
-        does not have any parameters,
+        $(P There can be only one destructor declared per class, although
+        other destructors $(DDSUBLINK spec/template-mixin, destructors, can be mixed in).
```

Comment:
Is this intentional behavior?

Reply:
Yes, but it was never written. Destructors of member variables are all called alongside the declared one, and a mixin template is a special kind of member. It is basically an embedded struct with its names exposed to the top level scope.